buildenv

```diff
@@ -0,0 +1,39 @@
+export ZOPEN_GIT_URL="https://gitlab.freedesktop.org/libbsd/libbsd"
+export ZOPEN_GIT_DEPS="libmd git make m4 autoconf perl libtool automake bash zoslib tar sed coreutils gnulib"
+#export ZOPEN_TARBALL_URL="https://libbsd.freedesktop.org/releases/libbsd-0.11.7.tar.xz"
+#export ZOPEN_TARBALL_DEPS="make libmd"
+export ZOPEN_TYPE="GIT"
+export ZOPEN_RUNTIME_DEPS="bash"
+export ZOPEN_COMP="clang"
+export ZOPEN_BOOTSTRAPD="./autogen"
+export ZOPEN_EXTRA_CFLAGS="-mzos-target=zosv2r4 -DNSIG=42 -D_ALL_SOURCE=1 -D_OPEN_SYS_FILE_EXT=1 -D_AE_BIMODAL=1 -D_ENHANCED_ASCII_EXT=0xFFFFFFFF -D_OPEN_SYS_SOCK_IPV6 -D_UNIX03_SOURCE -D_UNIX03_THREADS -D_XOPEN_SOURCE_EXTENDED=1 -D_POSIX_C_SOURCE=200112L -D_POSIX_SOURCE -DPOSIX_SIGNALS" 
```

Comment:
To reduce redundancies, can you remove `"-DNSIG=42 -mzos-target=zosv2r4 -D_ALL_SOURCE -D_AE_BIMODAL=1` etc since they are part of https://github.com/ZOSOpenTools/comp_clangport/blob/main/buildenv#L91C58-L91C72.

Reply:
Done